src/Downloader.ts

```diff
@@ -170,6 +170,7 @@ class Downloader {
     if (!forceRender) {
       //* Objects order in array matters!
       this.baseUrl = basicURLDirector.buildDownloaderBaseUrl([
+        { condition: await MediaWiki.hasWikimediaMobileRestApi(), value: MediaWiki.mobileRestApiUrl.href },
         { condition: await MediaWiki.hasWikimediaDesktopRestApi(), value: MediaWiki.desktopRestApiUrl.href },
```

Comment:
I don't really know why the `Rest` came in here?! It does not match the [official names on the wiki](https://github.com/openzim/mwoffliner/wiki/API-end%E2%80%90points)

I would realy want to keep the word `Rest` for [these API](https://www.mediawiki.org/wiki/API:REST_API)

Reply:
I removed `Rest` wording for desktop and mobile.